### PR TITLE
Add Geospatial data type example

### PR DESCRIPTION
**Pull request by bsbodden**

### CODE REVIEW

File: src/test/java/io/redis/examples/GeoExample.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
+//EXAMPLE: geo_tutorial
+package io.redis.examples;
+
+//REMOVE_START
```
Comment (sazzad16) [inline src/test/java/io/redis/examples/GeoExample.java:4]: it is not ended

Comment (bsbodden) [reply src/test/java/io/redis/examples/GeoExample.java:4]: Resolved

**Comment by uglide** [comment]
LGTM

**Review by uglide — COMMENTED** [review]
LGTM

**Pull request merged**